embedded-hal/CHANGELOG.md

```diff
@@ -65,7 +71,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 *** This is (also) an alpha release with breaking changes (sorry) ***
 
 ### Changed
-- The Minimum Supported Rust Version (MSRV) is now 1.60.0
+- The Minimum Supported Rust Version (MSRV) is now 1.59.0
```

Comment:
```suggestion
- The Minimum Supported Rust Version (MSRV) is now 1.60.0
```

Reply:
This is correct, I had inadvertently changed this with a `1.59`->`1.60` mass replace.

Reply:
I see, thanks!